src/app/personsok/PersonsokSkjema.tsx

```diff
@@ -28,6 +28,15 @@ interface Props {
 }
 const FormStyle = styled.article`
     padding: ${theme.margin.layout};
+    .skjemaelement__label {
+        margin-top: 0.5rem;
+        margin-bottom: 0rem;
+    }
+    .lenke {
+        .typo-normal {
+            margin-top: 1rem;
```

Comment:
Som hovedregel bør vi unngå styling som er avhengig av html-strukturen og klasser fra felles-biblioteket. Dette er fordi html-struktur endres på forløpende, og da er det fort gjort at stylingen ikke fungerer mer. Det samme problemet kan oppstå med bruk av felles-biblioteket sine klasse. De kan endres på uten at vi egentlig gjør noe. :O 

En annen sånn hovedregel er at vi prøver så langt det lar seg gjøre å bruke `margin-bottom` fremfor `margin-top`.
Dette har med hvordan browsere kollapser sammen marginer å gjøre, og vi har der funnet ut at det er lettere å forstå seg på hvordan ting fungerer om vi holder oss til `margin-top` og `margin-right`.

For `skjemaelement` så kunne vi f.eks gjort;
```
    .skjemaelement {
        margin-bottom: 0.5rem;
    }
    .skjemaelement__label {
        margin-bottom: 0rem;
    }
```

For mer luft ovenfor DREK-lenken, så kan vi legge til `margin-bottom: 1rem` i `SectionStyle` og få samme effekt. Men da unngår vi å forholde oss til html-strukturen i `LenkeDrek`.

Reply:
Takk for god tilbakemelding 😄